starlite/config/response_cache.py

```diff
@@ -14,6 +14,12 @@
     from starlite.types import CacheKeyBuilder
 
 
+class CACHE_FOREVER:  # noqa: N801
```

Comment:
Maybe, `NO_EXPIRY` instead?

Reply:
My thinking was that `CACHE_FOREVER` is unambigous even without context, i.e. if you have the value `CACHE_FOREVER` somewhere in your namespace, you'll know immediately that that it's related to caching and what it does. 

Does that make sense?